### DISCUSSION TITLE
Multi-Site Management

### DISCUSSION THREAD

**Discussion by leo79901** [apache/cloudstack#10089] [Q&A] [answered]
### Hi everyone,
On the QA website (https://qa.cloudstack.cloud/simulator/), I discovered that the management server is capable of **managing multiple zones** across different countries around the world. 
![image](https://github.com/user-attachments/assets/c9133d17-e7ba-41e6-bfe9-e62087d5f5f2)

Also, on the website, I came across a picture which further validates this.
![image](https://github.com/user-attachments/assets/58d57504-6f42-4389-bbec-0e196d452ec5)

Consequently, I understand that ACS offers multi-zone management. It's excellent news that we can administer several zones **via a single web interface**.

Nevertheless, at the same time, I have three questions:

1. Regarding the zones in different countries globally, the network connection between the management server and hypervisor hosts is extremely costly and only has a bandwidth of **5-10 Mbps**. Can it handle this?
2. The network the network connection between the management server and hypervisor is **unreliable**. Occasionally, it may be down for hours or even days. How will the zones that lose their connection to the management server function?
3. Is the configuration of managing multiple zones on different countries around the world by a management server cluster suitable for a **production environment**?

**Comment by DaanHoogland** [comment]
It seems that your conditions do not make it feasible to implement your desired architecture, @leo79901 .
- A host not being able to reache the management server for hours or days is not manageable. No alarms will be raised and no deployments, migrations or scheduled actions can be taken (snapshots, backups, scheduled stops starts.
- The low bandwith would not have to be a problem, but for some uploads.
- You can experiment with such a setup to determine if the intrinsic limitation are applicable for you, but..

In general the answer is no, this is not suitable for production.

cc @andrijapanicsb @alexandremattioli ??

**Comment by andrijapanicsb** [comment, marked as answer]
As down Daan wrote - you need stable connection between the main Zone (where the management servers are) and the remote zones (remote KVM hosts) - bandwidth is not that much of an issue (though, 5-10 Mbps is too low) - but you have to have a stable connections - otherwise your management plane (of remote Zones/hypervisors etc) is not possible (you can't e.g. start instance, attach volume etc - do ANY actions on VMs in those remote Zones)

**Comment by leo79901** [comment]
Thanks a lot! 
Now I know your means. For the production environment, the better way is to install management server clusters in each nation and develop a Cloud Management Platform to manage each cluster via API.
